spec/models/user/email_spec.rb

```diff
@@ -42,5 +42,16 @@
         expect(subject.errors[:email].count).to eq(1)
       end
     end
+
+    context 'when the email is not confirmed' do
+      subject { create(:user_email, :unconfirmed) }
+
+      it 'sends email with ActiveJob queue' do
+        message_delivery = instance_double(ActionMailer::MessageDelivery)
```

Comment:
can you link me to some articles as to why we need to patch Devise? I don't see anything nasty in their mail implementation that requires that.